zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -301,13 +297,13 @@ public Paragraph getLastParagraph() {
         Map<String, String> info = new HashMap<>();
         info.put("id", p.getId());
         info.put("status", p.getStatus().toString());
-        info.put("started", p.getDateStarted().toString());
-        info.put("finished", p.getDateFinished().toString());
+        info.put("started", p.getDateStarted() == null ? null : p.getDateStarted().toString());
```

Comment:
Could you please explain, what will `null` represent in the started of finished field of the `paragraphsInfo` and what are the cases when this might happen?

Will this create a new places in code, we need to check for null's now?